src/mixins/_forms.scss

```diff
@@ -164,3 +183,9 @@
     margin-right: 0;
   }
 }
+
+@mixin applyBorder($borderColor, $borderWidth, $borderStyle) {
```

Comment:
Nice work - i think let's put this mixin in `mixins.scss` as it can be useful with other components. Otherwise looks good to go